pkg/kptfile/kptfileutil/util.go

```diff
@@ -54,6 +55,21 @@ func WriteFile(dir string, k *kptfilev1.KptFile) error {
 	return nil
 }
 
+func WriteFileFS(dir paths.FileSystemPath, k *kptfilev1.KptFile) error {
```

Comment:
Please add godoc. Do we need FS in the name here ? Since FileSystemPath has FileSystem by default these methods can be without FS suffix.